### PR TITLE
[Jellyfin] Fix how refresh is handled

### PR DESCRIPTION
**Pull request by GiviMAD**
Hello,

This PR fixes/unifies how the refresh command is handled by the different channels.
Also include a minor upgrade of the Jellyfin sdk.

### CODE REVIEW

File: bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java [hunk 1]
```diff
@@ -586,17 +489,27 @@ private void handleMediaControlCommand(ChannelUID channelUID, Command command)
         }
     }
 
-    private void seekToPercentage(int percentage) throws SyncCallback.SyncCallbackError, ApiClientException {
+    private void seekToPercentage(Command command)
+            throws IllegalArgumentException, SyncCallback.SyncCallbackError, ApiClientException {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java:493]: You added `IllegalArgumentException`.
What is the code that can trigger this exception ?

File: bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java [hunk 2]
```diff
@@ -586,17 +489,27 @@ private void handleMediaControlCommand(ChannelUID channelUID, Command command)
...
         }
+        int percentage = Integer.parseInt(command.toFullString());
         var seekPositionTick = Math.round(((float) lastRunTimeTicks) * ((float) percentage / 100.0));
         logger.debug("Seek to {}%: {} of {}", percentage, seekPositionTick, lastRunTimeTicks);
         seekToTick(seekPositionTick);
     }
 
-    private void seekToSecond(long second) throws SyncCallback.SyncCallbackError, ApiClientException {
+    private void seekToSecond(Command command)
+            throws IllegalArgumentException, SyncCallback.SyncCallbackError, ApiClientException {
```
Comment (lolodomo) [inline bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java:508]: You added `IllegalArgumentException`.
What is the code that can trigger this exception ?

**Review by lolodomo — COMMENTED** [review]
2 questions

File: bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java (hunk 1 above)
Comment (GiviMAD) [reply bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java:493]: Ummm, yes maybe it wasn't the best decision.

The method could throw a NumberFormatException which extends from IllegalArgumentException.
As the parseItemUUID could also throws an IllegalArgumentException, I simplified it just adding common catch for the IllegalArgumentException to handle all as the same in the handleCommand method. But it's probably most accurate to have a "throws NumberFormatException" in both two methods.

Comment (GiviMAD) [reply bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java]: I have changed it to NumberFormatException in both methods.

Comment (GiviMAD) [reply bundles/org.openhab.binding.jellyfin/src/main/java/org/openhab/binding/jellyfin/internal/handler/JellyfinClientHandler.java]: I have added a specific catch for NumberFormatException.

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Comment by GiviMAD** [comment]
Thank you for the review!

**Pull request merged**